### ISSUE TITLE
Module compiled with Swift 4.2.1 cannot be imported by the Swift 5.0 compiler

### ISSUE THREAD

**Issue by jordanhbuiltbyhq** [apollographql/apollo-ios#478]
Please add support for Swift 5 with Xcode 10.2 :)

**Comment by uyphanha** [comment]
Got the same problem..

**Comment by aBuder** [comment]
Hey I found an workaround, if you use Cocoa pods try this.
The current master branch has fix, but is not released. See also the current commit 

https://github.com/apollographql/apollo-ios/pull/427/commits/40b8186af68a6dccf108ef98c9d82b1a8f204a63


`pod 'Apollo', :inhibit_warnings => true, :git => 'https://github.com/apollographql/apollo-ios.git', :branch => 'master'`

**Comment by uyphanha** [comment]
Thanks, will update my pod.

**Comment by SRozhina** [comment]
Cannot use ```pod 'Apollo', :inhibit_warnings => true, :git => 'https://github.com/apollographql/apollo-ios.git', :branch => 'master'``` because of using Apollo in custom module with Cocoapods. There is no way to add git path dependency to podspec. Could you please release this feature?

**Comment by indevizible** [comment]
it seems still error after execute script.
`Apollo.framework/check-and-run-apollo-cli.sh: line 71: npm: command not found`

**Comment by BeshBashBosh** [comment]
#427 seems to have fixed this on 30th Jan but no tagged release has occurred since then so will have to point to master in podfile until then.

**Comment by mike-marcacci** [comment]
Is there any known workaround for those of us using Carthage? Just upgraded XCode on total autopilot 🤦🏻‍♂️

**Comment by ernesto** [comment]
@SRozhina You can work with custom modules using the `s.dependecy 'Apollo'` on .podspec and overriding Apollo framework on the app podfile:
```
  # Internal Libraries
  pod 'ProjectCore', path: '../project-ios-sdk'
    
  # Network
  pod 'Apollo', :inhibit_warnings => true, :git => 'https://github.com/apollographql/apollo-ios.git', :branch => 'master'

```

**Comment by mike-marcacci** [comment]
OK, well... it took me WAY too long to figure this out, but I ended up simply going into the `Carthage/Checkouts/apollo-ios` directory, pulling, and building. I ended up using #475 instead of master.

Also make a note NOT to run `carthage update`, as it will upgrade to an incompatible version of Starscream and you'll chase your tail for a bit trying to get the build to work.

**Comment by martijnwalraven** [comment]
This should be fixed in 0.10.0.
